src/cms/templates/pages/page_form.html

```diff
@@ -182,27 +182,24 @@ <h2 class="heading font-normal">
                             </div>
                         {% endif %}
                     {% endif %}
-                    <label for="{{ page_translation_form.slug.id_for_label }}" class="block mb-2 font-bold">
-                        {{ page_translation_form.slug.label }}
-                        <button type="button" class="p-1 bg-gray-200 rounded ml-2"
-                            onclick='document.getElementById("id_slug").readOnly = false;'>
-                            <i data-feather="edit-2" class="inline-block"></i>
+                    <div id="link-container" class="flex items-center">
+                        <label for="{{ page_translation_form.slug.id_for_label }}" class="mr-2 font-bold">
+                            {{ page_translation_form.slug.label }}:
+                        </label>
+                        <div class="mr-2">
+                            {% spaceless %}
+                            <span>{{ page_link }}</span>
+                            {% render_field page_translation_form.slug|add_error_class:"slug-error" class="inline appearance-none rounded" size=page_translation_form.instance.slug|length %}
+                            {% endspaceless %}
+                        </div>
+                        <button id="edit-btn" type="button" class="mr-4">
+                            <i data-feather="edit-3"></i>
+                        </button>
+                        <button id="copy-btn" type="button">
+                            <i data-feather="copy"></i>
                         </button>
-                    </label>
-                    <div class="my-2 text-s text-gray-600">{{ page_translation_form.slug.help_text }}</div>
-                    <div id="slug-div" class="appearance-none w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400">
-                        {% spaceless %}
-                            <div style="display: table; white-space: nowrap;">
-                                <span style="display: table-cell;">https://integreat.app/{{ region.slug }}/{{ language.slug }}/</span>
-                                {% if page_translation_form.instance.ancestor_path %}
-                                    <span style="display: table-cell;">{{ page_translation_form.instance.ancestor_path }}/</span>
-                                {% endif %}
-                                <span class="w-full table-cell">
-                                    {% render_field page_translation_form.slug|add_error_class:"slug-error" class="appearance-none w-full rounded" %}
-                                </span>
-                            </div>
-                        {% endspaceless %}
                     </div>
+                    <div class="my-2 text-s text-gray-600">{{ page_translation_form.slug.help_text }}</div>
```

Comment:
I think the help text can be omitted here. Since the slug is updated automatically when the title changes, there is no need for the case what happens when the field is empty. And the users will notice that illegal characters are filtered out once they submit the form with an invalid slug.

```suggestion
```